### ISSUE TITLE
Exposing `base16` values to primary config

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [winneon/hypersixteen#3]
Is it possible to use `hypersixteen` to expose the base16 values of whatever theme is being used to the primary config-file?

For instance,

```js
  borderColor: base16.base0A,
```

**Comment by win0na** [comment]
Yes, that would be possible by pushing the initial `base16` object with the rest of the config. preferably at the beginning, right after [this line](https://github.com/winneon/hypersixteen/blob/master/src/HyperSixteen.js#L78).
